File: src/net/Defracted/AdminUtils/util/Formatters.java
package net.Defracted.AdminUtils.util;

import net.md_5.bungee.api.ChatColor;

public class Formatters {
    public static String chat(String str) {
        return ChatColor.translateAlternateColorCodes('&', str);
    }

    public static boolean isStrNum(String str) {
        try {
            Integer.parseInt(str);
            return true;
        } catch (Exception e) {
            return false;
        }
    }

    // next method
}

File: src/main/kotlin/net/defracted/adminutils/util/Formatters.kt
package net.defracted.adminutils.util

import net.md_5.bungee.api.ChatColor


object Formatters {
    fun chat(str: String?): String {
        return ChatColor.translateAlternateColorCodes('&', str)
    }

    fun isStrNum(str: String): Boolean {
        return try {
            str.toInt()
            true
        } catch (e: Exception) {
            false
        }
    }
}